### DISCUSSION TITLE
Custom VMA LibName

### DISCUSSION THREAD

**Discussion by jurganson** [amerkoleci/Vortice.Vulkan#44] [Ideas]
As per my previous feature request #43 I now wish we could do the same thing but instead for the VulkanMemoryAllocator. So instead of the hardcoded "vma" `LibName` string inside Vortice.Vulkan.Vma, perhaps we could setup our own DLL import string? 

Thanks a bunch for this awesome binding project!

**Comment by amerkoleci** [comment]
New version is comming with support for this (https://github.com/amerkoleci/Vortice.Vulkan/commit/c9b78b9087277f476562a5daec27e6e2e3aba84e)

Basically you attach your own resolve library using Vma.ResolveLibrary += ResolveLibraryEvent

Cheers!

**Reply by jurganson** [reply]
Awesome, thanks a lot again for the swift response!

**Comment by jurganson** [comment]
@amerkoleci I've began switching to `Vortice.VMA` however I need to specify the `LibName` for the dll import. 

ATM I am trying to switch to `Vortice.VMA` on iOS and I have a statically linked `.a` file, which I normally import using the extern P/Invoke methods, with a import signature of `[DllImport("__Internal")]`

However the `VmaDllImporterResolver` allows for overriding the entire import process - which I assume will only work for dynamic libraries? 

I see that the code generator has some functionality for adding `[DllImport` ... so I assume it will generate those stubs - but is there something I am missing for using them?
